faq/upgrade-faq.md

```diff
@@ -1,26 +1,42 @@
 ---
-title: FAQs After Upgrade
-summary: Learn about the FAQs after upgrading TiDB.
+title: FAQs Upgrade and After Upgrade
```

Comment:
How about moving "FAQs" to the end? Because it sounds a little bit weird to me. 🤔
```suggestion
title: Upgrade and After Upgrade FAQs
```